DOC-3574: Adding documentation for limitation related to nested json (#2930)
harness/developer-hub: docs/platform/20_References/runtime-inputs.md

For example, `<+input>.default(\'london,uk\').allowedValues(\'bengaluru,india\',\'newyork,usa\',\'london,uk\')`.

> JSON body as a string can also be used currently as the variable runtime input default value. 

Reviewer: ```suggestion
You can use JSON body as a string for the variable runtime input default value. 
```

NOTE: Currently we don’t support the use of nested JSON body as a string as the variable runtime input default value. 

> For example, this JSON string `<+input>.default('{\"risk\": 100,\"availabilityVsCost\": \"balanced\",\"drainingTimeout\": 120,\"lifetimePeriod\": \"days\", \"fallbackToOd\": true}')` can be used as a runtime input, but this JSON string `<+input>.default('{\"risk\": 100,\"availabilityVsCost\": \"balanced\",\"drainingTimeout\": 120,\"lifetimePeriod\": \"days\", \"fallbackToOd\": true,\"scalingStrategy\": {\"terminationPolicy\": \"default\" }}')` is currently not supported as a runtime input.

Reviewer: ```suggestion
For example, you can use the following JSON string for a runtime input.

`<+input>.default('{\"risk\": 100,\"availabilityVsCost\": \"balanced\",\"drainingTimeout\": 120,\"lifetimePeriod\": \"days\", \"fallbackToOd\": true}')` 

Harness does not support the following JSON string for a runtime input.

`<+input>.default('{\"risk\": 100,\"availabilityVsCost\": \"balanced\",\"drainingTimeout\": 120,\"lifetimePeriod\": \"days\", \"fallbackToOd\": true,\"scalingStrategy\": {\"terminationPolicy\": \"default\" }}')` 
```

For example, `<+input>.default(\'london,uk\').allowedValues(\'bengaluru,india\',\'newyork,usa\',\'london,uk\')`.

You can use JSON body as a string for the variable runtime input default value. 

:::info note

> Harness does not support the use of a nested JSON body as a string for the variable runtime input default value. 

Reviewer: ```suggestion
Harness does not support using a nested JSON body as a string for the variable runtime input default value. 
```
